sdk/src/clearingHouse.ts

```diff
@@ -819,10 +819,10 @@ export class ClearingHouse {
 		);
 	}
 
-	private async checkIfAccountExists(account: PublicKey) {
+	private async checkIfAccountExists(account: PublicKey): Promise<boolean> {
 		try {
 			const accountInfo = await this.connection.getAccountInfo(account);
-			return accountInfo && true;
+			return accountInfo != null;
```

Comment:
lol what the

Reply:
lol no clue but at least it wasnt being used